packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -190,11 +196,11 @@ class Metrics {
       origin: {
         buildType: 'prod',
         name: 'endpoint',
-        networkType: 'unknown',
+        networkType: 'unknown', // TODO: check if we can get mediaType
         userAgent: this.getUserAgentInfo(),
         clientInfo: {
-          clientType: options.clientType,
-          os: 'linux', // TODO: BUG AND DOESNT ACCEPT OSNAME ON CA -> bowser.osname || 'null'
+          clientType: options.clientType, // TODO: Only clientType: 'TEAMS_CLIENT' is whitelisted
+          os: 'linux' || this.osName(),
```

Comment:
The "or" case will never hit, needs to be swapped
```suggestion
          os: this.osName() || 'linux',
```

Reply:
ya currently we are hardcoding it and in the future we need to calculate and send . There is a specific set of values which they accept